### DISCUSSION TITLE
morallicensing1

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#28] [Announcements]
# morallicensing1

Exploring the concept of Moral Licensing and its implications in daily life and larger societal contexts.

https://nonzerosum.games/morallicensing1.html

**Comment by Nikko-A** [comment]
No idea how you managed to combine my interests for the psyche and geopolitics but kudos to you! I found your blog from a post on the Obsidian subreddit and I was interested in your work. Suffice to say, I am now going to be following this blog. 

That aside from Elon Musk was hilarious! While reading, I had some reservations on why he was included but that made it for a funnier and more effective example. Good work!

**Reply by jamesstephenbrown** [reply]
Thanks Nikko, very happy to have a like-mind on board, and an Obsidian fan too! The [next blog on moral licensing](https://nonzerosum.games/morallicensing2.html) is up now, which gets to an idea I'd had that originally inspired the post.

Yes, any use of Musk is made with hesitancy these days, his quote was too good not to use - though his lack of self-awareness in applying that very principle to himself appears to have no bounds...

Again, I'm very pleased to have you on board, please share your comments where you have them, I'm interested in dialogue and the ideas of others, as much as I am sharing my own.
